roles/openshift_cluster_autoscaler/files/role.yml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Role
+metadata:
+  name: cluster-autoscaler
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  resourceNames:
+  - cluster-autoscaler
+  - cluster-autoscaler-status
+  verbs:
+  - create
+  - get
+  - patch
+  - update
+  attributeRestrictions: null
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  attributeRestrictions: null
```

Comment:
Remove the :null entries

Reply:
The null entries allow this to be idempotent.  oc_obj compares the incoming with the in-memory.  The in-memory has `attributeRestrictions: null` and therefore recreates the objects each time.

We need to make the oc_obj more intelligent when comparing objects and verify the incoming objects are the same as the in-memory and not compare any fields that are not passed.

That work will need to come in separate PR.